### PR TITLE
 adds bug fixes and feature to 3.1.2 rel notes

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.8 only
- [Errata](https://errata.devel.redhat.com/docs/doc_text_info/101801)
- [Preview](http://file.del.redhat.com/asakthiv/winc-3.1.2-rel-notes/windows_containers/windows-containers-release-notes-3-x.html#wmco-3-1-2)  Requires VPN
- @mburke5678 please merge

### CODE REVIEW

File: windows_containers/windows-containers-release-notes-3-x.adoc [hunk 1]
```diff
@@ -21,13 +21,34 @@ ifndef::openshift-origin[]
...
+
+[id="wmco-3-1-2-win2022"]
+==== Windows Server 2022 support
+
+With this release, Windows Server 2022 now supports VMware vSphere.
+
+[id="wmco-3-1-2-bug-fixes"]
+=== Bug fixes
+
+* Previously, Windows Containers on Windows Nodes were assigned the wrong DNS Server IP. Therefore, the DNS resolution did not work. This bug fix removes the hard-coded cluster DNS information and parametrizes the value as a command-line argument.
```
Comment (aravindhp) [inline windows_containers/windows-containers-release-notes-3-x.adoc:49]: `Previously, Windows Containers on Windows Nodes were assigned the wrong DNS Server IP if a non-default cluster DNS was specified during the cluster install.`

**Review by kelbrown20 — COMMENTED** [review]
Looks great Amrita! 
/lgtm

**Pull request merged**